tests/sals/automated_chatflows/test_vdc_chatflows.py

```diff
@@ -12,6 +12,13 @@
 class VDCChatflows(VDCBase):
     flavor = "silver"
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls._import_wallet("vdc_init")
+        cls._import_wallet("grace_period")
+        j.core.config.set("VDC_INITIALIZATION_WALLET", "vdc_init")
```

Comment:
This needs to be checked first if it is in the config before changing it, then at tearDownClass restore its original value is exist